DXMainClient/DXGUI/Multiplayer/CnCNet/TunnelListBox.cs

```diff
@@ -122,6 +122,9 @@ private void TunnelHandler_TunnelsRefreshed(object sender, EventArgs e)
 
         private void TunnelHandler_TunnelPinged(int tunnelIndex)
         {
+            if (tunnelIndex >= ItemCount)
+                return;
```

Comment:
It actually happens on the first run when refreshing the tunnel list because `TunnelPinged` event handler is fired before `TunnelsRefreshed` and it tries to update non-existant tunnel. This is most likely a temporary workaround to make it work.

Reply:
Doesn't happen with fixed order now.